doc/nrf/protocols/matter/end_product/versioning.rst

```diff
@@ -0,0 +1,78 @@
+.. _ug_dfu_in_matter:
+
+Device Firmaware Update version configuration
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Device Firmware Update (DFU) is the procedure of upgrading the application firmware version on a device.
+You can transfer the updated images to the device in two ways:
+
+    Upgrade over Bluetooth LE
+
+    `Upgrade over Matter (Matter OTA)<https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/matter/overview/dfu.html>`_, where updates are sent through a wireless connection, like Bluetooth® Low Energy.
+
+Before creating and performing DFU, You need to preper proper versionig configuration.
+Versioning can be done by creating `VERSION file<https://docs.zephyrproject.org/latest/build/version/index.html>`_, or by chenging build configuration in prf.conf.
+
+Versioning with VERSION file
+****************************
+
+VERSION file is resposible for assignment values to MCUBoot version in the following way:
+"MAJOR.MINOR.PATCH.TWEAK"
+
+To use versioning with specified for this file, user needs to create in the application directory a file named VERSION with proper content:
+
+.. code-block:: console
+
+    VERSION_MAJOR =
+    VERSION_MINOR =
+    PATCHLEVEL =
+    VERSION_TWEAK =
+
+.. note::
+   It is crucial to assign any value to one of the variables, if not, VERSION file will be ignored and Kconfig values will be assgined to version of application.
+
+Versioning by changing configuration
+************************************
+
+.. _ug_matter_dfu_ota:
+
+Device Firmware Upgrade over Matter
+===================================
+
+In case of DFU over Matter user should eddit prj.conf file in the application directory.
+Following lines should be added to mentioned file to change Matter version:
+
+.. code-block:: console
+    CONFIG_CONFIG_CHIP_DEVICE_SOFTWARE_VERSION=<version number>
```

Comment:
<version number> and <version string> should be mentioned in a text before using. Also console code block doesn't seem to be correct here. You should better use :kconfig:option: before the Kconfig name.